src/i18n/en.json

```diff
@@ -0,0 +1,138 @@
+{
+  "common": {
+    "cancel": "Cancel",
+    "copyright_MD": "© {year} Doppler LLC. All rights reserved. [Privacy and Legal Policy](https://www.fromdoppler.com/en/legal/privacy-policy).",
+    "help": "Help",
+    "hide": "Hide",
+    "message": "Message",
+    "recaptcha_legal_HTML": "Site protected by reCAPTCHA and the Google <a href=\"https://policies.google.com/privacy?hl=en\">Privacy Policy</a> and <a href=\"https://policies.google.com/terms?hl=en\">Terms of Service</a>.",
+    "send": "Send",
+    "show": "Show"
+  },
+  "feature_panel": {
+    "email_automation": "Email automation",
+    "email_automation_description": "Right person, right message, right time",
+    "email_automation_remarks": "Send 100% personalized Emails according to the behavior and interests of your Subscribers. Save time and money!",
+    "email_editor": "Emails Editor",
+    "email_editor_description": "Crea Emails en minutos y accede a nuestra Galería de Plantillas",
+    "email_editor_remarks": "Nuestras Plantillas para Email son totalmente Responsive y fácilmente editables desde nuestro Editor HTML.",
+    "forms": "subscription forms",
+    "forms_description": "Add new contacts to your Lists using custom Forms",
+    "forms_remarks": "Classics and pop-ups with Single or Double Opt-In subscription. You decide how you want them to look, what data to request and where to place them!"
+  },
+  "footer": {
+    "iso": "ISO Quality Certification 9001:2008"
+  },
+  "forgot_password": {
+    "back_login": "Did you remember your password and want to return?",
+    "back_login_after_forgot": "Return to login",
+    "confirmation_message_HTML": "<p>Check your inbox!</p><p> You’ll find an Email with steps to follow.</p>",
+    "description": "Do not worry! It happens to everyone.",
+    "description2": "Enter your email and we will help you"
```

Comment:
`Email`

Reply:
@carodipietro esto que comenta @cbernat no es parte de los cambios de @GusBaamonde, pero me parece que los comentarios de Cecilia son correctos, ¿Crees que deberíamos aplicar esta sugerencia?

Reply:
@cbernat is right.